common/environment/setup/install.sh

```diff
@@ -258,3 +258,34 @@ _vmkdir() {
 		install -dm${mode} ${_destdir}/${dir}
 	fi
 }
+
+_vcompletion() {
+	local cmd="$1" shell="$2" file="$3"
+	local _bash_completion_dir=usr/share/bash-completion/completions/
+	local _fish_completion_dir=usr/share/fish/vendor_completions.d/
+	local _zsh_completion_dir=usr/share/zsh/site-functions/
+
+	if [ -z "$DESTDIR" ]; then
+		msg_red "$pkgver: vcompletion: DESTDIR unset, can't continue...\n"
+		return 1
+	fi
+
+	if [ -z "$cmd" -o -z "$shell" ]; then
+		msg_red "$pkgver: vcompletion: command or shell argument unset.\n"
```

Comment:
Please make it separate errors.

Reply:
Better yet make message like of other vinstall-based functions.

Reply:
Copied one, thanks! Much better :p